extensions/README.md

```diff
@@ -0,0 +1,44 @@
+# [Gardener Extensions Library](https://gardener.cloud)
+
+![Gardener Extensions Logo](../logo/gardener-extension-180px.png)
+
+Project Gardener implements the automated management and operation of [Kubernetes](https://kubernetes.io/) clusters as a service. Its main principle is to leverage Kubernetes concepts for all of its tasks.
+
+Recently, most of the vendor specific logic has been developed [in-tree](https://github.com/gardener/gardener). However, the project has grown to a size where it is very hard to extend, maintain, and test. With [GEP-1](https://github.com/gardener/gardener/blob/master/docs/proposals/01-extensibility.md) we have proposed how the architecture can be changed in a way to support external controllers that contain their very own vendor specifics. This way, we can keep Gardener core clean and independent.
+
+This directory contains utilities functions and common libraries meant to ease writing the actual extension controllers.
+Please consult https://github.com/gardener/gardener/tree/master/docs/extensions to get more information about the extension contracts.
+
+## Known Extension Implementations
+
+Check out these repositories for implementations of the Gardener Extension contracts:
+
+| Infrastructure Provider | DNS Provider | Operating System | Container Runtime | Network Plugin | Generic Extensions |
```

Comment:
I think that the table is not the most convenient way to present similar list of know repositories - there is no common thing that would match the row and the column. 
I would suggest something like the existing list. 

```suggestion
### Infrastructure Provider

- [Alicloud](https://github.com/gardenergardener-extension-provider-alicloud) - optional description goes here
- [AWS](https://github.com/gardenergardener-extension-provider-aws)
- [Azure](https://github.com/gardenergardener-extension-provider-azure)
- [GCP](https://github.com/gardenergardener-extension-provider-gcp)
- [MetalStack](https://github.com/metal-podgardener-extension-provider-metal)
- [OpenStack](https://github.com/gardenergardener-extension-provider-openstack)
- [Packet](https://github.com/gardenergardener-extension-provider-packet)
- [vSphere](https://github.com/gardenergardener-extension-provider-vsphere)

### DNS Provider

- [External DNS Management](https://github.com/gardener/external-dns-management) [*]

<sub>[*] Alicoud DNS, AWS Route53, Azure DNS, Cloudflare DNS, Google CloudDNS, OpenStack Designate</sub>

### Operating System

- [CoreOS/FlatCar](https://github.com/gardener/gardener-extension-os-coreos)
- [CoreOS/FlatCar (Alicloud)](https://github.com/gardener/gardener-extension-os-coreos-alicloud)
- [GardenLinux](https://github.com/gardener/gardener-extension-os-gardenlinux)
- [Debian/Ubuntu (MetalStack)](https://github.com/metal-pod/os-metal-extension)
- [Ubuntu](https://github.com/gardener/gardener-extension-os-ubuntu)
- [Ubuntu (Alicloud)](https://github.com/gardener/gardener-extension-os-ubuntu-alicloud)
- [SuSE JeOS](https://github.com/gardener/gardener-extension-os-suse-jeos)

### Container Runtime

- [gVisor](https://github.com/gardener/gardener-extension-runtime-gvisor)

### Network Plugin

- [Calico](https://github.com/gardener/gardener-extension-networking-calico)
- [Cilium](https://github.com/gardener/gardener-extension-networking-cilium)

### Generic Extensions

- [Shoot Certificate Service](https://github.com/gardener/gardener-extension-shoot-cert-service)
- [Shoot DNS Service](https://github.com/gardener/gardener-extension-shoot-dns-service)
```